lib/menu_displayer.rb

```diff
@@ -0,0 +1,18 @@
+class MenuDisplayer
+
+  require_relative "./menu.rb"
+
+  attr_reader :menu
+
+  def initialize
```

Comment:
Curious if it would be worth injecting menu as an argument here?

Reply:
still a bit hazy on dependencies, what are the merits of initialising it?

Reply:
Same here! In this case one small merit could be refactoring the method so its one line shorter:

`def initialize(menu = Menu.new)
    @menu = menu
  end`

I could be a small win, but a win all the same!